docs/configuration.md

```diff
@@ -225,13 +231,23 @@ target_match:
   [ <labelname>: <labelvalue>, ... ]
 target_match_re:
   [ <labelname>: <regex>, ... ]
+  
+# Matchers that have to be fulfilled in the alerts to be muted 
+# with support of negative matching. 
```

Comment:
As above: "A list of matchers that have to be fulfilled by the target alerts to be muted."